cmake/modules/FindLibSSH.cmake

```diff
@@ -34,6 +34,7 @@ if (NOT NoSSH)
           /sw/include
           ${CMAKE_INCLUDE_PATH}
           ${CMAKE_INSTALL_PREFIX}/include
+          {LIBSSH_INCLUDE}
```

Comment:
should be `${LIBSSH_INCLUDE}`, or not?

Further I would put those at the top (first place), because they might be used to overwrite other locations if multiple version of libssh are in the system. Same below for the library path...